src/components/organisms/Section/Section.js

```diff
@@ -0,0 +1,21 @@
+import React, { useContext } from "react";
+import { CryptoApiContext } from "providers/CryptoApiProvider";
+import ArticleWrapper from "components/molecules/ArticleWrapper/ArticleWrapper";
+import { Loading } from "components/atoms/Loading/Loading";
+import { Wrapper } from "./Section.styles";
+
+const Section = () => {
+  const { articles } = useContext(CryptoApiContext);
+
+  return (
+    <Wrapper>
+      {articles.length > 1 ? <div>Articles</div> : <Loading />}
```

Comment:
pasowałoby Ci ogarnąć jeszcze empty state i error state, bo w tym przypadku jak articles będzie puste to będzie infinite spinner

najlepiej jest wyciągać logikę poza rendera, dlatego nawet taki drobny zapis jak `articles.length > 1` wyciągnąłbym poza returna i jakoś fajnie nazwał, np:


```
/* zakładamy że masz już error i loading state z contextu */

const showContent = articles.length && !error && !loading;
const showLoader = loading && !error;
const showNoContent = articles.length === 0 && !loading && !error;
const showError = articles.length === 0 && !loading && error;

return (
  <div>
     {showLoading && <Spinner />} 
     {showNoContent && <p>No content</p>}
     {showError && <p>Error</p>}
     {showContent && articles?.map(article => <ArticleWrapper>...</ArticleWrapper>)}
  </div>
)
```